dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/udp/udp.vue

```diff
@@ -251,6 +251,19 @@
       max-height: 600px;
       overflow-y: scroll;
       padding:0 20px;
+
+      ::selection {
+        background: #409EFF ;
+        color: white;
+      }
+      ::-moz-selection {
```

Comment:
Please remove `::-moz-selection` and `::-webkit-selection`

Reply:
Thanks @chengshiwen, done.